<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>setting</class>
 <widget class="QWidget" name="setting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Setting</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>47</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>kthresh</string>
   </property>
  </widget>
  <widget class="QSlider" name="sl_kthresh">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>291</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="ch_bRemoveBlobs">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>320</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>bRemoveBlobs</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="ch_bMorphology">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>320</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>bMorphology</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="gr_Morphology">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>190</y>
     <width>271</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>morphology</string>
   </property>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>61</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>modeMorpho</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>47</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>iterations</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ksizeMorpho</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ln_iterations">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>70</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ln_ksizeMorphology">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>70</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbb_modeMorphology">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>140</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gr_RemoveBlobs">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>40</y>
     <width>271</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>removeBlobs</string>
   </property>
   <widget class="QComboBox" name="cbb_methodFindcontours">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>140</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>47</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>width</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ln_height">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>70</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>-1,-1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>47</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>area</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ln_area">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>100</y>
      <width>70</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>-1,-1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>47</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>height</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>methodFindcontours</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ln_width">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>70</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>-1,-1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>modeFindcontours</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbb_modeFindcontours">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>140</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gr_Blur">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>251</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>blur</string>
   </property>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>61</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>modeBlur</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>47</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>ksizeBlur</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ln_ksizeBlur">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>70</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbb_modeBlur">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>140</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="ch_bBlur">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>bBlur</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_kthresh">
   <property name="geometry">
    <rect>
     <x>396</x>
     <y>5</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="gr_Blur_2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>251</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>thresh</string>
   </property>
   <widget class="QComboBox" name="cbb_methodAdaptive">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>140</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>methodAdaptive</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbb_modeThresh">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>140</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>61</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>modeThresh</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>47</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>blockSize</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ln_blockSize">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>70</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>11</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>47</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ln_C">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>70</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
  </widget>
  <widget class="QRadioButton" name="rad_thresh">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>thresh</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="rad_adaptive">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>adaptive</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rad_otsu">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>150</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>otsu</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but_OK">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>320</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but_Cancel">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>320</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
